src/main/kotlin/no/nav/familie/ef/iverksett/økonomi/simulering/SimuleringService.kt

```diff
@@ -64,4 +69,22 @@ class SimuleringService(
             oppsummering = simuleringsresultatDto
         )
     }
+
+    private fun hentSimuleringsresultatOgFiltrerPosteringer(
+        utbetalingsoppdrag: Utbetalingsoppdrag,
+        stønadType: StønadType
+    ): DetaljertSimuleringResultat {
+        val fagOmrådeKoder = fagområdeKoderForPosteringer(stønadType)
+        val simuleringsResultat = oppdragKlient.hentSimuleringsresultat(utbetalingsoppdrag)
+        return simuleringsResultat.copy(
+            simuleringsResultat.simuleringMottaker
+                .map {
+                    it.copy(
```

Comment:
ikke trivielt å lese hva `it` er her heller - så kan med fordel rename den til `simuleringsmottaker`

Reply:
evt. mottaker

Reply:
Renamet til mottaker